sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CreateNewCaseSteps.java

```diff
@@ -93,6 +118,14 @@ public CreateNewCaseSteps(WebDriverHelpers webDriverHelpers, CaseService caseSer
           webDriverHelpers.waitUntilElementIsVisibleAndClickable(EditCasePage.REPORT_DATE_INPUT);
           webDriverHelpers.clickOnWebElementBySelector(CASE_SAVED_POPUP);
         });
+      Then(
```

Comment:
already in place, but at some point, I would like to refactor and keep the logic as declared here (since now is within creation/update methods/logic)